src/components/Calculator.js

```diff
@@ -1,187 +1,106 @@
-          type="button"
-          className="char"
-          onClick={this.showResult}
-        >
-          3
-        </button>
-        <button
-          value="+"
-          type="button"
-          className="char orange"
-          onClick={this.showResult}
-        >
-          +
-        </button>
-        <button
-          value="0"
-          type="button"
-          className="zero"
-          onClick={this.showResult}
-        >
-          0
-        </button>
-        <button
-          value="."
-          type="button"
-          className="char"
-          onClick={this.showResult}
-        >
-          .
-        </button>
-        <button
-          value="="
-          type="button"
-          className="char orange"
-          onClick={this.showResult}
-        >
-          =
-        </button>
-      </div>
-    );
+  let result = 0;
+  if (obj !== null) {
+    if (obj.next !== null) {
+      result = obj.next;
+    } else result = obj.total;
   }
-}
+  return (
+    <div className="Calculator">
+      <span className="input">{result}</span>
+      <button type="button" value="AC" className="char" onClick={showResult}>
+        AC
+      </button>
+      <button value="+/-" type="button" className="char" onClick={showResult}>
+        +/-
+      </button>
+      <button value="%" type="button" className="char" onClick={showResult}>
+        %
+      </button>
+      <button
+        value="÷"
+        type="button"
+        className="char orange"
+        onClick={showResult}
+      >
+        ÷
+      </button>
+      <button type="button" value="7" className="char" onClick={showResult}>
+        7
+      </button>
+      <button value="8" type="button" className="char" onClick={showResult}>
+        8
+      </button>
+      <button value="9" type="button" className="char" onClick={showResult}>
+        9
+      </button>
+      <button
+        value="x"
+        type="button"
+        className="char orange"
+        onClick={showResult}
+      >
+        x
+      </button>
+      <button value="4" type="button" className="char" onClick={showResult}>
+        4
+      </button>
+      <button value="5" type="button" className="char" onClick={showResult}>
+        5
+      </button>
+      <button value="6" type="button" className="char" onClick={showResult}>
+        6
+      </button>
+      <button
+        value="-"
+        type="button"
+        className="char orange"
+        onClick={showResult}
+      >
+        -
+      </button>
+      <button value="1" type="button" className="char" onClick={showResult}>
+        1
+      </button>
+      <button value="2" type="button" className="char" onClick={showResult}>
+        2
+      </button>
+      <button value="3" type="button" className="char" onClick={showResult}>
+        3
+      </button>
+      <button
+        value="+"
+        type="button"
+        className="char orange"
+        onClick={showResult}
+      >
+        +
+      </button>
```

Comment:
- [ ] When an operator button is clicked multiple times consecutively, it throws a browser error. Kindly apply error handling method or condition and make sure your app is free of any errors. Kindly make changes. :pray: 

![Screenshot from 2021-09-16 16-50-36](https://user-images.githubusercontent.com/64894927/133599850-a2b1ea04-77d7-4bb1-97e2-7168fcce5eda.png)